src/routes/api/text/[language_code]/+server.ts

```diff
@@ -9,7 +9,12 @@ export const GET: RequestHandler = async ({ url, params }): Promise<Response> =>
 		const speech_language_code = SpeechLanguageCode.create(params.language_code)
 
 		const text_db = new TextDb()
-		const texts = await text_db.find_many(speech_language_code)
+
+		const limit = url.searchParams.get('limit')
```

Comment:
Make Limit value object

Reply:
Limit value object: Check if null and write tests. It should always be more than 1. Can use valid ID.